compiler/control/OMROptions.cpp

```diff
@@ -2693,8 +2694,8 @@ OMR::Options::jitPreProcess()
             {
             if (_aggressivenessLevel != -1) // -1 means not set
                {
-               if (OMR::Options::isAnyVerboseOptionSet())
-                  TR_VerboseLog::writeLine(TR_Vlog_INFO, "_aggressivenessLevel=%d; must be between 0 and 5; Option ignored", _aggressivenessLevel);
+               if (OMR::Options::isAnyVerboseOptionSet()) 
+                     TR_VerboseLog::writeLineLocked(TR_Vlog_INFO, "_aggressivenessLevel=%d; must be between 0 and 5; Option ignored", _aggressivenessLevel);
```

Comment:
Too many leading spaces on this line

Reply:
@TurnipXenon just one minor stylistic change to make and we can merge this PR. Thanks for contributing this!

Reply:
We'll fix them up another time. I don't want a space change to halt up this PR getting merged.